bom/pom.xml

```diff
@@ -74,6 +77,16 @@
                 <artifactId>slf4j-api</artifactId>
                 <version>${slf4j-api.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot</artifactId>
+                <version>${spring-boot.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-core</artifactId>
+                <version>${spring.version}</version>
+            </dependency>
```

Comment:
Doesn't this come transitively with Spring Boot? I'd try to define the Spring Boot version only.

Reply:
we have here a show-stopper: this library is compiled for Java 8, but Spring requires at Java 17.

Reply:
@Johannes-Schneider Do you see a possibility to increase the Java version of this library to 17? AFAIK CAP, Cloud SDK and XSUAA - which are the main stakeholders of this library at the moment - are all on Java 17 already.

Reply:
+ @MatKuhr: Any opinion here?

Reply:
I think we can increase to Java 17 as minimum and indicate this a bit by increasing to `0.20.0` or something like that. Since it's not a stable version and most users (if not all) are on 17 already, that should be okay. But probably better to do in a separate PR.

On a side note, I would consider to leave out the spring version in the BOM, because this library doesn't need very frequent updates, and the spring version here will fall behind easily. And since the spring module is optional and newly introduced I'd expect that dependency conflicts are rather unlikely for the foreseeable future 😉

Reply:
I think what @MatKuhr meant was that we should not add the Spring dependency to the BOM, but only to the newly introduced `sap-spring-properties` module. There we can of course reference a version directly.

Reply:
my bad, sorry